File: Python/8py 1/boolean_to_string.py
"""
|--------------------------------------------------------------------------
| Convert a Boolean to a String
|--------------------------------------------------------------------------
|
"""

def boolean_to_string(b):
   return 'True' if b==True else 'False'


def boolean_to_string(b):
   return str(b)


def boolean_to_string(b):
   return ('False', 'True')[b]


def boolean_to_string(b):
   return f"{b}"


print(boolean_to_string(True), "True")
print(boolean_to_string(False), "False")

File: PHP/8php 1/booleanToString.php
<?php
/*
|--------------------------------------------------------------------------
| Convert a Boolean to a String
|--------------------------------------------------------------------------
|
*/

function booleanToString($b) {
   return $b == true ? "true" : "false";
}


function booleanToString($b) {
   return $b ? "true" : "false";
}


function booleanToString($b) {
   return $b == false ? "false" : "true";
}


print_r(booleanToString(true) . ' ' . 'true' . '
');
print_r(booleanToString(false) . ' ' . 'false' . '
');